src/examples/chat/client.cc

```diff
@@ -30,7 +30,7 @@ int main(int argc, char* argv[]) {
   zmq::context_t context(1);
 
   std::vector<std::tuple<server_address_t, client_address_t, nickname_t>>
-      connect_tuple = {{server_address, client_address, nickname}};
+      connect_tuple = std::vector<std::tuple<server_address_t, client_address_t, nickname_t>>{std::tuple<server_address_t, client_address_t, nickname_t>{server_address, client_address, nickname}};
```

Comment:
Wow. I had no idea clang++ and g++ were so different. I would change it to this then:

```cpp
std::vector<std::tuple<...>> connect_tuple = {std::make_tuple(server_address, client_address, nickname)};
```